src/plugins/console/public/application/containers/editor/legacy/console_editor/keyboard_shortcuts.ts

```diff
@@ -63,4 +65,30 @@ export function registerCommands({
       senseEditor.moveToNextRequestEdge(false);
     },
   });
+
+  coreEditor.registerKeyboardShortcut({
+    name: 'gotoline',
+    keys: { win: 'Ctrl-L', mac: 'Command-L' },
+    fn: (editor) => {
+      const line = parseInt(prompt('Enter line number') ?? '', 10);
+      if (!isNaN(line)) {
+        editor.gotoLine(line);
+      }
+    },
+  });
+}
+
+export function unregisterCommands(senseEditor: SenseEditor) {
+  const coreEditor = senseEditor.getCoreEditor();
+  const commands = [
+    'send to Elasticsearch',
+    'open documentation',
+    'auto indent request',
+    'move to previous request start or end',
+    'move to next request start or end',
```

Comment:
nit: any ideas about moving it into constants? any then use them both for `register` and `unregister`.